doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1881,6 +1881,59 @@ PSA Crypto support
          - Supported
          - Supported
 
+
+MCUboot bootloader
+==================
+
+.. toggle::
```

Comment:
Should be moved above or below Security Features support as a separate section, as already suggested by Divya

```
MCUboot bootloader
******************
```